mycli/main.py

```diff
@@ -582,6 +583,34 @@ def handle_clip_command(self, text):
             return True
         return False
 
+    def handle_prettify_binding(self, text):
+        try:
+            statements = sqlglot.parse(text)
```

Comment:
i don't know much about this library, but i assume it's mysql based, so you'll want to specify read="mysql"

and when you call sql(), write="mysql"

Reply:
Hey thanks!  Added, substituting `dialect="mysql"` for `write=`.

Although actually `read="mysql"` makes `parse()` fail on the example I was using, simplified:

```
python -c 'import sqlglot; print(sqlglot.parse_one("""SELECT "2021-01-01" + interval 1 month""", read="mysql").sql(dialect="mysql"))'
…
sqlglot.errors.ParseError: Invalid expression / Unexpected token. Line 1, Col: 21.
  SELECT "2021-01-01" + interval 1 month
  ```

`python-dateutil` is installed if that matters.  Should I file an issue?

Reply:
yea you can file an issue, it might be because you're using double quotes around the date